docs/docs/faq.md

```diff
@@ -80,6 +80,14 @@ Please note, many keyboards only have a single PCB which includes the “brains
 
 Currently, ZMK only supports wireless split, but wired split is possible and we welcome contributions!
 
+### How is the latency?
+
+The latency of ZMK is comparable to other firmware offerings. ZMK is equipped with a variety of scanning methods and [debounce algorithms](https://zmk.dev/docs/features/debouncing) that can affect the final measured latency. [This video](https://www.youtube.com/watch?v=jWL4nU-vtWs) shows a latency comparison of ZMK and other keyboard firmwares.
```

Comment:
Because this is an internal page, we'd use a relative link to the file, like so:

```suggestion
The latency of ZMK is comparable to other firmware offerings. ZMK is equipped with a variety of scanning methods and [debounce algorithms](features/debouncing.md) that can affect the final measured latency. [This video](https://www.youtube.com/watch?v=jWL4nU-vtWs) shows a latency comparison of ZMK and other keyboard firmwares.
```

You can see some other examples of this on [intro.md](https://raw.githubusercontent.com/zmkfirmware/zmk/a197bf88bab8a3e4d60846212b684fb54c7e5290/docs/docs/intro.md).